lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -296,6 +296,8 @@ def add_ip(xml, options)
       def add_auth_purchase_params(xml, options)
         xml.ssl_dynamic_dba                     options[:dba] if options.has_key?(:dba)
         xml.ssl_merchant_initiated_unscheduled  merchant_initiated_unscheduled(options) if merchant_initiated_unscheduled(options)
+        xml.ssl_get_token                       recurring_transaction(options) if recurring_transaction(options)
+        xml.ssl_add_token                       recurring_transaction(options) if recurring_transaction(options)
```

Comment:
I couldn't find `ssl_get_token` in the Zendesk tickets, but I think this is where my hang up is. I was just assuming it would be the same as the `ssl_add_token`.

Reply:
For posterity since we chatted outside of the ticket, the ssl_get_token was requested specifically in the Jira ticket, but the ssl_add_token performs that function as well as store so it was removed.